src/Stratis.Bitcoin/P2P/Peer/NetworkPeerServer.cs

```diff
@@ -180,7 +180,7 @@ public void Listen()
             }
             catch (Exception e)
             {
-                this.logger.LogError("Exception occurred: {0}", e.ToString());
+                this.logger.LogDebug("Exception occurred: {0}", e.ToString());
```

Comment:
Just a quick explanation why do we expect a disposed exception?

Reply:
What about catching an ObjectDisposedException?

Reply:
https://github.com/stratisproject/StratisBitcoinFullNode/issues/1141#issuecomment-365255037

Reply:
See that comment. We can probably catch that specific exception, But the problem that the documentation of the function - https://msdn.microsoft.com/en-us/library/system.net.sockets.tcplistener.accepttcpclientasync%28v=vs.110%29.aspx - doesn't mention it, probably because it is just a side effect of closing the socket and not a feature of this specific method and anyway ask whether or not there can be another side effect leading to slightly different exception type and you have no guarantees that if you catch just one type that another type can be thrown because of the very same thing again